packages/visualizations/src/components/Map/types.ts

```diff
@@ -95,31 +95,14 @@ export interface ChoroplethDataValue {
     label?: string;
 }
 
-export enum ChoroplethShapeTypes {
-    GeoJson = 'geojson',
-    VectorTiles = 'vtiles',
-}
-
-/** `ChoroplethShapeValue` implementation based on a GeoJSON FeatureCollection.  */
```

Comment:
The type `ChoroplethShapeValue` doesn't exist anymore. This comment is obsolete.

Reply:
Thanks, I totally forgot about that one !